src/Hazelcast.Net/DistributedObjects/Impl/HMultiMap.cs

```diff
@@ -289,22 +289,22 @@ public async Task ClearAsync()
 
         /// <inheritdoc />
         public Task LockAsync(TKey key)
-            => LockAsync(key, Timeout.InfiniteTimeSpan);
+            => LockAsync(key, LeaseTime.MaxValue);
 
         /// <inheritdoc />
         public Task<bool> TryLockAsync(TKey key)
-            => TryLockAsync(key, TimeSpan.Zero, Timeout.InfiniteTimeSpan);
+            => TryLockAsync(key, TimeToWait.Zero, LeaseTime.MaxValue);
 
         /// <inheritdoc />
         public Task<bool> TryLockAsync(TKey key, TimeSpan timeToWait)
-            => TryLockAsync(key, timeToWait, Timeout.InfiniteTimeSpan);
+            => TryLockAsync(key, timeToWait, LeaseTime.MaxValue);
```

Comment:
Java uses `Long.MAX_VALUE` instead of -1 (`LeaseTime.MaxValue`)

Reply:
Never mind, after looking at the implementation, `-1` seems like the correct value